src/kirby/helpers/color-helper.ts

```diff
@@ -1,23 +1,21 @@
 declare var require: any;
 export class ColorHelper {
   static style: any = require('sass-extract-loader!./color-helper.scss');
-
-  public static getThemeColor(name: string): RgbColor {
-    const color = ColorHelper.style.global['$kirby-colors'].value[name];
-    return color ? color.value : undefined;
+  public static getThemeColor(name: string) {
+    return ColorHelper.getColor(name);
+  }
+  public static getContrastColor(name: string) {
+    return ColorHelper.getColor(name + '-contrast');
   }
-
   public static getThemeColorRgbString(name: string) {
     return ColorHelper.getRgbString(ColorHelper.getThemeColor(name));
   }
-
   public static getRgbString(color: RgbColor) {
     if (!color) {
       return undefined;
     }
     return `rgb(${color.r}, ${color.g}, ${color.b})`;
   }
-
   public static getAlphaIn255Range(alpha: number): number {
```

Comment:
Revert deleted spacing:
```suggestion

    public static getAlphaIn255Range(alpha: number): number {
```